frontend/packages/console-app/src/components/quick-starts/controller/QuickStartTaskHeader.tsx

```diff
@@ -88,7 +88,7 @@ const QuickStartTaskHeader: React.FC<QuickStartTaskHeaderProps> = ({
         </>
       }
       step={taskIndex}
-      onNavItemClick={onTaskSelect}
+      onNavItemClick={() => onTaskSelect(taskIndex - 1)}
       navItemComponent="Button"
```

Comment:
This does not seem supported. I get a React error:

![image](https://user-images.githubusercontent.com/8126518/88825468-9d714580-d195-11ea-894a-515e690b971d.png)

Reply:
That's a patternfly component warning. Not related to code that i have.

Reply:
Oh I got it now. I think it's the prop that is being passed on. It should be `button` instead of `Button`. Again leftover from older implementation.

Reply:
Yup, understood. We can leave it or fix it, doesn't make a big difference.